sample-apps/segmentation/lib/train.py

```diff
@@ -92,14 +121,16 @@ def train_post_transforms(self, context: Context):
             AsDiscreted(
                 keys=("pred", "label"),
                 argmax=(True, False),
-                to_onehot=True,
-                n_classes=2,
+                to_onehot=(True, True),
+                n_classes=len(self.label_names),
```

Comment:
I guess this shall be len(self.label_names) + 1? to include background..

Reply:
The background is included in the dictionary label_names